plugin/vcs/github/github.go

```diff
@@ -155,9 +174,104 @@ func (p *Provider) FetchUserInfo(ctx context.Context, oauthCtx common.OauthConte
 	return p.fetchUserInfo(ctx, oauthCtx, fmt.Sprintf("users/%s", username))
 }
 
+func getRoleAndMappedRole(roleName string) (githubRole RepositoryRole, bytebaseRole common.ProjectRole) {
+	// Please refer to https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#repository-roles-for-organizations
+	// for the detailed role descriptions of GitHub.
+	switch roleName {
+	case "admin":
+		return RepositoryRoleAdmin, common.ProjectOwner
+	case "maintain":
+		return RepositoryRoleMaintain, common.ProjectOwner
+	case "write":
+		return RepositoryRoleWrite, common.ProjectOwner
+	case "triage":
+		return RepositoryRoleTriage, common.ProjectDeveloper
+	case "read":
+		return RepositoryRoleRead, common.ProjectDeveloper
+	}
+	return "", ""
+}
+
 // FetchRepositoryActiveMemberList fetch all active members of a repository
-func (p *Provider) FetchRepositoryActiveMemberList(ctx context.Context, oauthCtx common.OauthContext, instanceURL, repositoryID string) ([]*vcs.RepositoryMember, error) {
-	return nil, errors.New("not implemented yet") // TODO: https://github.com/bytebase/bytebase/issues/928
+func (p *Provider) FetchRepositoryActiveMemberList(ctx context.Context, oauthCtx common.OauthContext, _, repositoryID string) ([]*vcs.RepositoryMember, error) {
+	var allCollaborators []RepositoryCollaborator
+	page := 1
+	for {
+		collaborators, hasNextPage, err := p.fetchPaginatedRepositoryCollaborators(ctx, oauthCtx, repositoryID, page)
+		if err != nil {
+			return nil, errors.Wrap(err, "fetch paginated list")
+		}
+		allCollaborators = append(allCollaborators, collaborators...)
+
+		if !hasNextPage {
+			break
+		}
+		page++
+	}
+
+	allMembers := make([]*vcs.RepositoryMember, len(allCollaborators))
```

Comment:
```suggestion
	var allMembers []*vcs.RepositoryMember
```

Reply:
Don't need to optimize.

Reply:
Why do we not want to do this when we know the _exact_ length and capacity we want?

Reply:
Addressed in https://github.com/bytebase/bytebase/pull/1775/commits/b4d8c80280084e8cd0dcc4f286ee51019b9d042e